internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -1011,7 +1010,7 @@ func (c *ClusterManager) buildImagePullSecret(namespace string) *k8sCoreV1.Secre
 			Kind:       "Secret",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      imagePullSecretName,
+			Name:      kafkaConstants.ImagePullSecretName,
 			Namespace: namespace,
 		},
 		Type: k8sCoreV1.SecretTypeDockercfg,
```

Comment:
Not part of this PR of course, but should this be `SecretTypeDockerConfigJson` instead? Just based on the secret key name of `image-pull.dockerconfigjson`.

Reply:
Probably yes, and `SecretTypeDockerConfigJson` is the new format. So we should be using it instead.  

I am not sure we'd want to change it in this PR though.

Reply:
> I am not sure we'd want to change it in this PR though.

Agreed - thanks for confirming though